src/PhpSpec/Console/Command/DescribeCommand.php

```diff
@@ -64,9 +67,52 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $container = $this->getApplication()->getContainer();
         $container->configure();
 
-        $classname = $input->getArgument('class');
-        $resource  = $container->get('locator.resource_manager')->createResource($classname);
+        if ($input->getArgument('class')) {
+            $classname = $input->getArgument('class');
+        } else {
+            $questionHelper = $this->getApplication()->getHelperSet()->get('question');
+            $question = new Question('<info>Enter class to describe: </info>');
+
+            $question->setAutocompleterValues($this->getNamespaces());
+            $classname = $questionHelper->ask($input, $output, $question);
+        }
+
+        $resource = $container->get('locator.resource_manager')->createResource($classname);
 
         $container->get('code_generator')->generate($resource, 'specification');
     }
+
+    /**
+     * Get suites namespaces.
+     *
+     * @return array
+     */
+    private function getNamespaces()
```

Comment:
As this method becomes more complex maybe it makes sense to introduce a separate class (or even interface) like `NamespacesAutocompleteProvider` and cover it with tests. What do you think?